ci/lint/05_before_script.sh

```diff
@@ -6,4 +6,4 @@
 
 export LC_ALL=C
 
-git fetch --unshallow
+git fetch
```

Comment:
why?

Reply:
On Cirrus-CI, I received:

`fatal: --unshallow on a complete repository does not make sense`